types/react-text-marquee/tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all these rules from here.. New definitions cannot disable any rule and need to fix the errors if any